covenant.cabal

```diff
@@ -76,11 +76,19 @@ common test-lang
     -with-rtsopts=-N
 
   build-depends:
+    -- temporary, maybe, for debugging tests
```

Comment:
I'm pretty sure I made this comment but I can't remember what it is referring to. I'm _pretty damn sure_ that we're not going to be dropping QuickCheck from the _test suite_ so it can probably be removed.